lib/barbeque/executor/hako.rb

```diff
@@ -41,6 +119,41 @@ def env_options(envs)
           "--env=#{key}=#{value}"
         end
       end
+
+      def s3_key_for_stopped_result(hako_task)
+        "#{@s3_prefix}/#{hako_task.task_arn}/stopped.json"
+      end
+
+      def s3_client
+        @s3_client ||= Aws::S3::Client.new(region: @s3_region, http_read_timeout: 5)
+      end
+
+      def get_stopped_result(hako_task)
+        object = s3_client.get_object(bucket: @s3_bucket, key: s3_key_for_stopped_result(hako_task))
+        JSON.parse(object.body.read)
```

Comment:
I'm just curious why we rescue JSON parse error in `#extract_task_info` but do not in this method.